client/actions/Modals/index.js

```diff
@@ -10,24 +10,22 @@ import {
  * @param {string} modalName The name of the modal to set.
  * @param {!Object} modalProps The props to pass through the modal being set.
  */
-export const setModal = (modalName, modalProps) => dispatch => (
+export const setModal = (modalName, modalProps) => dispatch =>
```

Comment:
Move the returning `dispatch` to the next line

Reply:
I did idiot